core/modules/system/system.menu.inc

```diff
@@ -164,6 +177,11 @@ function system_menu_block_build(array $config) {
         $data['content']['#attached']['library'][] = array('system', 'smartmenus');
       }
     }
+    if (!empty($config['toggle']) && $config['toggle'] == TRUE) {
+      $data['content']['#wrapper_attributes']['data-menu-toggle-hash'] = $config['hash'];
+      $data['content']['#attached']['css'][] = backdrop_get_path('module', 'system') . '/css/menu-toggle.theme.css';
+      $data['content']['#prefix'] = theme('menu_toggle', array('enabled' => $config['toggle'], 'text' => 'Menu', 'hash' => $config['hash']));
```

Comment:
The word "Menu" should be translated on the way in here, rather than leaving it to the theme function.